yeat/Snakefile

```diff
@@ -13,7 +13,7 @@ import pandas as pd
 
 rule all:
     input:
-        expand("analysis/quast/{sample}/report.html", sample=config["sample"]),
+        expand("analysis/quast/{assemblers}/{sample}_report.html", assemblers=config["assemblers"], sample=config["sample"]),
```

Comment:
The first wildcard in this string makes more sense as `{assembler}` rather than `{assemblers}`. It helps to remember that a wildcard a placeholder that can take on a single value at a time. So a single `{assembler}`, populated with values from `config["assemblers"]` (which *is* appropriate in the multiple tense/cardinality).

Reply:
Understood!